metadata-ingestion/src/datahub/sql_parsing/tool_meta_extractor.py

```diff
@@ -29,7 +37,7 @@ def _get_last_line(query: str) -> str:
 
 
 @dataclass
-class ToolMetaExtractorReport(Report):
+class ToolMetaExtractorReport(SourceReport):
```

Comment:
Why is this change needed ?

Reply:
This was added as a result of **report.warning()**